src/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+require('./styles/layout.scss');
+require('./styles/theme.scss');
+
+// Requiring after app.scss ensures correct selector precedence
+require('./styles/icons.css');
+
+// Require icons
```

Comment:
We should remove all of these icons from this PR - or make an issue about replacing them once we have one for this project.

Reply:
@katjam i had world with @Nikomus about it we will remove it later but again i will double check with him